pilot/pkg/model/service.go

```diff
@@ -388,6 +392,10 @@ func (si *ServiceInstance) GetLocality() string {
 	if si.Endpoint.Locality != "" {
 		return si.Endpoint.Locality
 	}
+	if strings.Contains(si.Labels[LocalityLabel], k8sSeparator) {
+		// replace "." with "/"
```

Comment:
Isn't this exactly the opposite of the title for this PR? Should you be replacing "/" with "." instead?

Reply:
We need it to be `a/b/c` but k8s doesn't allow this, so they provide `a.b.c` and we will convert it to `a/b/c` for internal use.

Reply:
@howardjohn sure that's fine ... it was just confusing given that thee title of the PR indicates the opposite: 

```
Locality label istio-locality in k8s should not contain `/`, use `.`
```

@hzxuzhonghu mind updating the title of this PR (and your commit) to match the logic?

Reply:
Sure